.github/workflows/run-tests.yml

```
@@ -24,6 +24,11 @@ jobs:
 
       - name: Install poetry
         run: python -m pip install poetry
+
+      - name: Set poetry installer config
+        if: runner.os == 'macOS'
+        run: echo "POETRY_INSTALLER_NO_BINARY=:all:" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:44: Double quote to prevent globbing and word splitting

(shellcheck)